peachjam/js/components/DocumentContent/ShareSelectionModal.vue

```diff
@@ -46,6 +46,13 @@
               @click="modal.hide()"
             ><i class="bi bi-facebook facebook-forecolor share-icon" />
             </a>
+            <a
+              :href="`https://www.linkedin.com/sharing/share-offsite/?url=${ combined }`"
```

Comment:
It looks like linkedin is a bit like facebook in that it only allows a URL, not text + url (which is what Twitter supports and is what is in `combined`). So this should instead be `url=${ encodeURIComponent(url) }`